src/main/cpp/hw/DragonSparkFlex.cpp

```diff
@@ -187,12 +203,15 @@ bool DragonSparkFlex::IsForwardLimitSwitchClosed()
 
 bool DragonSparkFlex::IsReverseLimitSwitchClosed()
 {
+    if (m_spark->GetReverseLimitSwitch(m_reverseType).Get())
+        m_encoder.SetPosition(0);
```

Comment:
Do we also want to zero the encoder when we hit the reverse limit switch?

Reply:
I think this makes sense, if you hit the bottom of some moving part, that is 0? What do you propose, move this to the mech? I am okay with that too

Reply:
Yeah bro